iet.so/src/iet_worker.cpp

```diff
@@ -219,7 +212,7 @@ bool IETWorker::do_iet_power_stress(void) {
     rvs::action_result_t action_result;
     RSMI_POWER_TYPE type = RSMI_INVALID_POWER;
 
-    std::thread t(&IETWorker::blasThread, this, gpu_device_index, matrix_size_a, iet_ops_type, start, run_duration_ms,
+    std::thread t(&IETWorker::blasThread, this, gpu_device_index, matrix_size, iet_ops_type, start, run_duration_ms,
```

Comment:
good catch

Reply:
but what happens if we specify martix sizes a b and c, how can we ignore it ?

Reply:
then this is a todo as a,b,and c can be used to override behavior and there is no need to have a b and c to be of same dimension